e2e/tests-dfx/ledger.bash

```diff
@@ -210,6 +210,10 @@ tc_to_num() {
   assert_match "Refunded at block height"
   assert_match "with message: Subnet $SUBNET_ID does not exist"
 
+  CANISTER_ID="aaaaa-aa"
+  assert_command dfx ledger create-canister --amount=100 --next-to "$CANISTER_ID" "$(dfx identity get-principal)"
+  assert_match "unable to determine subnet: Canister is not assigned to any subnet"
+
```

Comment:
Can the test also ensure that the funds have not been spent?

Reply:
done